data/crac-file/crac-io-json/src/test/java/com/farao_community/farao/data/crac_io_json/NetworkActionImportExportTest.java

```diff
@@ -0,0 +1,395 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.data.crac_io_json;
+
+import com.farao_community.farao.data.crac_api.ActionType;
+import com.farao_community.farao.data.crac_api.NetworkElement;
+import com.farao_community.farao.data.crac_impl.AbstractRemedialAction;
+import com.farao_community.farao.data.crac_impl.remedial_action.network_action.*;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+import static com.farao_community.farao.data.crac_io_json.RoundTripUtil.roundTrip;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Joris Mancini {@literal <joris.mancini at rte-france.com>}
+ */
+public class NetworkActionImportExportTest {
+    @Test
+    public void pstSetpointJsonCreator() {
+        PstSetpoint pstSetpoint = new PstSetpoint(
+            "pstSetpointId",
+            "pstSetpointName",
+            "RTE",
+            new ArrayList<>(),
```

Comment:
Don't you want to create a constructor that doesn't need a UsageRule as argument?

Reply:
alright